### PR TITLE
feat: OIDC4vci in wallet - initiate issuance & authorize

### LINKED ISSUE #1588: OIDC VC Issuance - Mock Issuer : initiate issuance

### LINKED ISSUE #1589: [oidc4vci]  Mock Issuer : handle authorize request

### LINKED ISSUE #1629: [oidc4vci] - wallet UI to support initiate issuance, authorize
- added UI routers to handle incoming initiate issuance and send authorize request to issuer by reading issuer OIDC configuration

### PR DESCRIPTION
**Pull request by sudeshrshetty**
Added basic implementation for initiate issuance & authorize flow.
Updated mock issuer to support initiate issuance, authorize & login

- Closes #1588
- Closes #1589
- Closes #1629

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: scripts/run_ui_automation.sh [hunk 1]
```diff
@@ -64,17 +64,17 @@ healthCheck demo-hydra https://demo-hydra.trustbloc.local:7777/.well-known/openi
...
+#	cd $ROOT/test/fixtures/wallet-web
+#	docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml logs  --no-color >& docker-compose.log
+#
+#	exit 1
+#fi
+#
+#echo "stopping containers..."
+#cd $ROOT/test/fixtures/wallet-web
+#
+#(source .env && docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml down --remove-orphans)
```
Comment (rolsonquadras) [inline scripts/run_ui_automation.sh:80]: Looks like this was pushed by mistake.

Comment (sudeshrshetty) [reply scripts/run_ui_automation.sh]: done

File: test/mock/adapter/adapter.go [hunk 2]
```diff
@@ -411,6 +435,176 @@ func (v *adapterApp) oidcShareCallback(w http.ResponseWriter, r *http.Request) {
...
+		CredentialManifests:   []byte(credManifest),
+	}, "", "	")
+	if err != nil {
+		handleError(w, http.StatusInternalServerError,
+			fmt.Sprintf("failed to prepare issuer wellknown configuration : %s", err))
+
+		return
+	}
+
+	v.store.Put(key, issuerConf)
```
Comment (rolsonquadras) [inline test/mock/adapter/adapter.go:461]: Don't we need error check here ?

**Review by rolsonquadras — APPROVED** [review]

File: test/mock/adapter/adapter.go (hunk 2 above)
Comment (sudeshrshetty) [reply test/mock/adapter/adapter.go:461]: fixed

File: cmd/wallet-web/src/router/routes.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ export default [
     ],
   },
   {
-    path: 'oidc',
+    path: '/oidc',
```
Comment (talwinder50) [inline cmd/wallet-web/src/router/routes.js:64]: Just curious , why we need / here in this path only ?

**Review by talwinder50 — APPROVED** [review]

**Pull request merged**